README.adoc

```diff
@@ -0,0 +1,167 @@
+:toc:
+:toc-placement!:
+
+// gEmoji for admonitions, see
+// https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74#admonitions
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+// Links (alphabetical order)
+:apache_license: http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+:application_monitoring_operator: https://github.com/integr8ly/application-monitoring-operator[application-monitoring-operator]
+:export_policy: https://aerogear.org/legal/export.html[AeroGear Export Policy]
+:aerogear_freenode: irc://irc.freenode.net/aerogear[#aerogear on FreeNode IRC]
+:aerogear_jira: https://issues.jboss.org/projects/AEROGEAR/issues[AeroGear on JBoss Jira]
+:aerogear_matrix: https://matrix.to/#/!IipcvbGVqkiTUQauSC:matrix.org[#aerogear:matrix.org on Matrix]
+:mailing_list: https://groups.google.com/forum/#!forum/aerogear[Google Groups Mailing List]
+:minishift: https://github.com/minishift/minishift[Minishift]
+:rh_product_security: https://access.redhat.com/security/team/contact[Red Hat Product Security team]
+
+= Mobile Developer Console Operator
+
+ifdef::status[]
+.*Project health*
+image:https://travis-ci.com/aerogear/mobile-developer-console-operator.svg?branch=master[Build Status (Travis), link=https://travis-ci.com/aerogear/mobile-developer-console-operator.svg?branch=master]
+image:https://img.shields.io/:license-Apache2-blue.svg[License (License), link=http://www.apache.org/licenses/LICENSE-2.0]
+endif::[]
+
+
+The Mobile Developer Console Operator for Kubernetes provides an easy way to
+install and manage an AeroGear Mobile Developer Console Operator on Kubernetes.
+
+toc::[]
+
+
+== Limitations
+
+// https://issues.jboss.org/browse/AEROGEAR-9162
+[NOTE]
+====
+This operator currently only works on OpenShift. This is because it
+provisions `Route`s and also relies on the fact that OpenShift
+provides an OAuth server. In future we aim to make it work on vanilla
+Kubernetes also.
+====
+
+== Minishift Pre-requisites
+
+. Enabled CORS addon on Minishift	
+# https://github.com/aerogear/mobile-developer-console#enable-cors-in-the-openshift-cluster	
```

Comment:
This comment looks weird when Asciidoc is rendered: https://github.com/aerogear/mobile-developer-console-operator/pull/8/files?short_path=7bd4a92#diff-7bd4a925c695c2eb0eced3872b9b965f

Reply:
Would it be better to make it a link?

Reply:
I think so

Reply:
Should the user follow the steps at this link or is it just for further information only?

Reply:
User should follow the steps if they're on Minishift.

Reply:
Thanks @aliok I will include that and make this a link and then merge this one.